Sources/MacroToolkitExamplePlugin/AddAsyncMacroCore.swift

```diff
@@ -1,11 +1,13 @@
-import SwiftSyntax
 import MacroToolkit
+import SwiftSyntax
 import SwiftSyntaxMacros
 
 // Modified from: https://github.com/DougGregor/swift-macro-examples/blob/f61ac7cdca8dc3557e53f86e7e03df1353908d3e/MacroExamplesPlugin/AddAsyncMacro.swift
 
 enum AddAsyncMacroCore {
-    static func expansion(of node: AttributeSyntax?, providingFunctionOf declaration: some DeclSyntaxProtocol) throws -> DeclSyntax {
+    static func expansion(
+        of node: AttributeSyntax?, providingFunctionOf declaration: some DeclSyntaxProtocol
```

Comment:
put these wrapped arguments on separate lines (just makes it a bit more skim readable in my opinion, i should configure swift format to do that automatically when wrapping arguments)

Reply:
I looked at the configuration for swift-format and didn't find anything. Honestly though, I've always used Nick Lockwood's version.